src/common/buffer.cc

```diff
@@ -1809,6 +1809,9 @@ using namespace ceph;
     _buffers.splice(_buffers.begin(), bl._buffers );
     bl._len = 0;
     bl.last_p = bl.begin();
+
+    // update the last_p point to the first ptr
```

Comment:
this comment is a translation of following statement. so i don't think it helps. if you really want to document it. 

```c++
// we modified _buffers
```
would suffice.